diff --git a/src/combat/Actions.cpp b/src/combat/Actions.cpp
--- a/src/combat/Actions.cpp
+++ b/src/combat/Actions.cpp
@@ -901,7 +901,9 @@
 Action Actions::UpgradeAllCardsInHand() {
     return {[=] (BattleContext &bc) {
         for (int i = 0; i < bc.cards.cardsInHand; ++i) {
-            bc.cards.hand[i].upgrade();
+            if (bc.cards.hand[i].canUpgrade()) {
+                bc.cards.hand[i].upgrade();
+            }
         }
     }};
 }
